tests/dependencies/indirect/BUILD

```diff
@@ -1,19 +1,19 @@
 load("@rules_scala_annex//rules:register_toolchain.bzl", "register_zinc_toolchain")
 load("@rules_scala_annex//rules:scala.bzl", "scala_library")
-load("@rules_scala_annex//rules/scala:workspace.bzl", "scala_2_13_version")
+load("@rules_scala_annex//rules/scala:versions.bzl", "scala_2_13_version")
```

Comment:
It feels weird to have versions come from annex and then the jars come from the test workspace, but perhaps that is just how things are now?

Reply:
Hmm, it does. Would you prefer we split `versions.bzl` into two files: one for `rules_scala_annex` and another for `rules_scala_annex_test`? I figured that even though we're forced to declare the versions in multiple places and define separate dependencies trees, it would still be best to have a single source of truth.

Reply:
Yeah probably best to minimize the number of places versions can come from and leave it as is.